app/views/curation_concerns/file_sets/upload/_script_templates.html.erb

```diff
@@ -1,8 +1,7 @@
-<!-- TODO: make the upload_set_id available on the presenter -->
-<% if false && presenter.upload_set_id %>
+<% if @presenter && @presenter.upload_set_id %>
```

Comment:
I know this view is rendered when the user clicks Upload a file without a work (e.g. click upload form the homepage) and it does not seem that `@presenter` is set there. 

But there is a presenter set when a the user is viewing a work (i.e. GenericWork show).